calico_containers/integrations/kubernetes/calico_kubernetes.py

```diff
@@ -9,6 +9,7 @@
 
 # Append to existing env, to avoid losing PATH etc.
 # Need to edit the path here since calicoctl loads client on import.
+print('Setting up Env')
```

Comment:
Don't think the Env logging here is useful.  We already have logs which print out the ENV variables which we care about, and that is what is important.